client/src/boot/BootRoutes.tsx

```diff
@@ -23,24 +26,18 @@ class BootRoutes {
    * @param {Object} store Redux store
    * @param {Object} client The Apollo client
    */
-  constructor(store, client) {
-    this.store = store;
-    this.client = client;
-
+  constructor(private store: Store, private client: ApolloClient<any>) {
     // pageRouter must be initialised, regardless of whether we are
     // using page.js routing for this request.
     const base = Config.get('absoluteBaseUrl');
     pageRouter.setAbsoluteBase(base);
-
-    this.handleBeforeRoute = this.handleBeforeRoute.bind(this);
-    this.handleBeforeUnload = this.handleBeforeUnload.bind(this);
   }
 
-  setStore(store) {
+  public setStore(store: Store) {
```

Comment:
Your class method in TS can be defined as private, public or protected. This allows you to make clear what is internal logic to your class and what is meant for public consumption by other classes.